persistence-modules/pom.xml

```diff
@@ -75,6 +75,7 @@
         <!-- <module>spring-mybatis</module> --> <!-- needs fixing in BAEL-9021 -->
         <module>spring-persistence-simple</module>
         <module>spring-persistence-simple-2</module>
+        <module>apache-bookkeeper</module><!-- BAEL-2322 -->
```

Comment:
Let's keep the modules in alphabetical order.

Reply:
Done